CHANGES

```diff
@@ -1,8 +1,70 @@
 CHANGES - changes for swtpm
 
 version 0.9.0:
+  Note: The SElinux policy for swtpm was completely redone. For systems
+        with an SELinux policy the same policy as used in Fedora >= 40 is
```

Comment:
suggestion: which version of SELinux is now required

Reply:
`>= 40.17`